asllib/doc/asl.spec

```diff
@@ -3136,16 +3136,20 @@ typing relation get_for_constraints(
   math_layout = [_,_],
 };
 
-semantics relation eval_stmt(env: envs, s: stmt) -> Returning((vs: list0(native_value), new_g: XGraphs), new_env: envs)
-    | Continuing(new_g: XGraphs, new_env: envs) | TThrowing | TDynError | TDiverging
+semantics relation eval_stmt(env: envs, s: stmt) ->
+    | Continuing(new_g: XGraphs, new_env: envs)
+    | TReturning
+    | TThrowing
+    | TDynError
+    | TDiverging
 {
    prose_description = "evaluates a statement {s} in an environment {env},
                         resulting in one of four types of configurations (see
                         more details in
                         \secref{KindsOfSemanticConfigurations}):
                         \begin{itemize}
-                        \item returning configurations with values {vs}, execution graph {new_g}, and a modified environment {new_env};
```

Comment:
Just to double check: the variable names that were introduced here (`vs`, `new_g`, `new_env`) are defined somewhere else right?